### ISSUE TITLE
Expand mapping options

### ISSUE THREAD

**Issue by AnnCorp** [datasciencecampus/sdg-indicators#2791]
Currently limited to Local Authority maps only and need to expand this to include:
Towns and Cities boundaries
Police Force areas
Middle Layer Output Areas (MSOAs)
Need to consider how feed through what boundary to use and what API to pull it from.
Any solution to be easily expandable - will add other boundaries in future.
In some cases this will involve merging multiple files to produce a UK map (eg re Towns and Cities will pull in Eng & Wales boundary, with one from NI and another from Scotland).
Will provide more details re APIs and sample indicators

**Comment by AnnCorp** [comment]
Top priority is adding Towns and Cities eg for https://sustainabledevelopment-uk.github.io/11-6-2/ data already in tool (with geo codes). @norric1 is getting sign-off from ONS Geography but would be looking to combine:
England and Wales: http://geoportal.statistics.gov.uk/datasets/major-towns-and-cities-december-2015-boundaries?geometry=-8.452%2C50.927%2C2.002%2C52.123
Scotland Settlements and Localities: https://www.nrscotland.gov.uk/statistics-and-data/statistics/statistics-by-theme/population/population-estimates/special-area-population-estimates/settlements-and-localities
Northern Ireland Settlement Development Limits: https://www.nisra.gov.uk/support/geography/urban-rural-classification
Other boundaries will need to cover:
Police Force Area:
http://geoportal.statistics.gov.uk/datasets/police-force-areas-december-2016-full-clipped-boundaries-in-england-and-wales
Output Areas (note - might use MSOA/LSOA instead):
http://geoportal.statistics.gov.uk/datasets/output-area-december-2011-full-clipped-boundaries-in-england-and-wales 
@norric1 will add pointers to indicator examples when we have data.

**Comment by AnnCorp** [comment]
@shenavall pushing this ticket into the sprint so can start exploring how tackle this (happy to break ticket down to smaller jobs if needed). Can discuss tomorrow...

**Comment by Norric1Admin** [comment]
Dependent on https://github.com/ONSdigital/sdg-indicators/issues/2920

**Comment by AnnCorp** [comment]
Lucy will do a test with one of the maps to see if gets same issue as Chris.

**Comment by Norric1Admin** [comment]
@LucyGwilliamAdmin Options for an additional map on the NRP...

onsdigital.github.io/sdg-indicators/2-3-1/ (possible but Units are a problem)
onsdigital.github.io/sdg-indicators/3-a-1/ (best match overall) **!**
onsdigital.github.io/sdg-indicators/4-2-1/ (England-only, but a good backup)
onsdigital.github.io/sdg-indicators/9-1-1/ (possible but silly data)

**Comment by Norric1Admin** [comment]
Embed test (story map): http://sdg.mango-solutions.com/test-data-deploy/4-2-1/
Map test: http://sdg.mango-solutions.com/test-data-deploy/3-a-1/

**Comment by Norric1Admin** [comment]
This can't proceed until we get to the bottom of https://github.com/ONSdigital/sdg-indicators/issues/2944

**Comment by AnnCorp** [comment]
@LucyGwilliamAdmin - for us to discuss with Data team?

**Comment by LucyGwilliamAdmin** [comment]
@AnnCorp agreed - not sure if they still want to show maps at city/town level - I think functionality to add lower levels is already there
